frontend/device/generic/device.lua

```diff
@@ -140,6 +138,7 @@ function Device:resume()
     end
     self.screen_saver_mode = false
     self.powerd:refreshCapacity()
+    self.powerd:wakeUp()
```

Comment:
Oh, yes, git can help to maintain the dependency, I do not need to change koreader again after koreader-base commitment.

And the name of two powerd functions are not so clear, I may want to change them into, beforeSuspend and afterResume. What do you think?